### PR TITLE
Python generators: console_scripts entry points require setuptools

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/rpm-software-management/rpm/issues/664

### CODE REVIEW

File: scripts/pythondistdeps.py [hunk 1]
```diff
@@ -191,6 +191,11 @@
                             py_deps[name].append(spec)
                 if not dep.specs:
                     py_deps[name] = []
+            # console_scripts/gui_scripts entry points need pkg_resources from setuptools
+            if ('setuptools' not in py_deps and
+                    (dist.get_entry_map('console_scripts') or
+                        dist.get_entry_map('gui_scripts'))):
+                py_deps['setuptools'] = []
```
Comment (ignatenkobrain) [inline scripts/pythondistdeps.py]: shouldn't that be `python{dist.py_version}dist(setuptools)`?

Comment (hroncok) [reply scripts/pythondistdeps.py]: yes it should. omg

Comment (hroncok) [reply scripts/pythondistdeps.py]: amended

**Comment by ignatenkobrain** [comment]
Looks good, but I'll test it later today as well (I know one of packages which need it).

**Comment by hroncok** [comment]
Oh, I only noticed the PR number now :smiling_imp:

**Pull request merged**